tests/unit/utils.test.ts

```diff
@@ -131,8 +145,8 @@ describe("Parse Functions", () => {
         pubkey: "z6MkmzRwg47UWQxczLLLFfkEwpBGitjzJ1vKPE8U9ymd6fz6",
       },
     },
-  ])("parseNid", ({ input, expected }) => {
-    expect(utils.parseNid(input)).toEqual(expected);
+  ])("parseNodeId", ({ input, expected }) => {
+    expect(utils.parseNodeId(input)).toEqual(expected);
```

Comment:
Could we add a unit test for `parseRepositoryId`?

Reply:
243e8e3b3d23221e06f96c3a0a8dad363bc90458